daemon/src/maker_cfd.rs

```diff
@@ -311,6 +313,23 @@ impl Actor {
         self.cfd_feed_actor_inbox
             .send(load_all_cfds(&mut conn).await?)?;
 
+        let latest_announcement = self
+            .latest_announcement
+            .to_owned()
+            .context("Unaware of oracle's latest announcement.")?;
+
+        let own_address = ctx
+            .address()
+            .expect("actor to be able to give address to itself");
+        oracle::watch_attestation(
```

Comment:
I was very unsure about starting this task here. Maybe we should wait until after setting up the CFD?